moveit_ros/benchmarks/src/BenchmarkExecutor.cpp

```diff
@@ -40,7 +40,10 @@
 #include <tf2_eigen/tf2_eigen.h>
 
 #include <boost/regex.hpp>
+// TODO: Remove if boost > 1.72 (https://github.com/boostorg/timer/issues/12)
```

Comment:
```suggestion
// TODO: Remove if boost >= 1.72 (https://github.com/boostorg/timer/issues/12)
```
https://github.com/boostorg/timer/commit/c221a60102ed618a82116f84407de3f1713aa10f is part of 1.72